aten/src/ATen/native/TensorAdvancedIndexing.cpp

```diff
@@ -853,6 +897,191 @@ TORCH_IMPL_FUNC(index_add_cpu_out)
   }
 }
 
+void index_reduce_func_impl(
```

Comment:
Since we want to consider these prototypes or beta let's [add a TORCH_WARN_ONCE](https://github.com/pytorch/pytorch/wiki/PyTorch's-Python-Frontend-Backward-and-Forward-Compatibility-Policy#prototypes-beta-and-stable-parts-of-pytorch) before we forget.